src/styles/css/choices.css

```diff
@@ -356,4 +356,9 @@
   opacity: .5;
 }
 
+.choices[data-type*="select-one"] .choices__input.is-hidden,
+.choices[data-type*="select-multiple"] .choices__input.is-hidden {
+  display: none;
```

Comment:
I think `.is-hidden` should be set here as a util: https://github.com/AndyRN/Choices/blob/ab9a471517015598e65ed4ebe7e5ac874dbd3cf9/src/styles/scss/base.scss#L123

Reply:
Happy to include the `is-hidden` class within the base style sheet if you'd prefer?

Just to note, it still needs to be set here as well to reach the required level of specificity. As you can see here, it is being outranked by the more specific style set later on.

![image](https://user-images.githubusercontent.com/1238199/33880913-0fc79a18-df2b-11e7-9485-26a5fe42ab80.png)

Which is why, in this case for the multi-select, the full `.choices[data-type*="select-multiple"] .choices__input.is-hidden` selector needs to be specified.